newdle/templates/joined_email.txt

```diff
@@ -0,0 +1,10 @@
+Hi,
+
+{{ participant }} replied to "{{ title }}".
+
+Selected dates:
+{% for item in answers %}
+    {{ item }}
+{% endfor %}
+
+You can get more info using the {{ summary_link }} summary link
```

Comment:
This doesn't look particularly good in a plaintext mail. What about something like this?

```
Check all results on the summary page:
{{ summary_link }}
```